Rug.Domain.Tests/ChampionshipCreatorTests.cs

```diff
@@ -0,0 +1,70 @@
+namespace Rug.Domain.Tests
+{
+    using NUnit.Framework;
+    using Rug.Domain.Championship;
+    using Rug.Domain.Services;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class ChampionshipCreatorTests
+    {
+        [Test]
+        public void CreateMatches_WhenTeamsAreOddNumber_ShouldReturnOnlyValidMatches()
+        {
+            // Arrange
+            var draw = new Draw(new Dictionary<int, byte>()
+            {
+                // [drawNumber] = teamId
+                [1] = 1, [2] = 2, [3] = 3, [4] = 4, [5] = 5
+            });
+
+            // Act
+            var matches = ChampionshipCreator.CreateMatches(draw).ToArray();
+
+            // Assert
```

Comment:
Do you like writing tests with constrained based model (Assert.That)? I think its more powerful and flexible and closer to human language.